<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleDialog</class>
 <widget class="QDialog" name="RoleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Custom Role</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/img/hammer.png</normaloff>:/img/hammer.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="aspects_tab">
      <attribute name="title">
       <string>Aspects</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTreeView" name="tree_aspects"/>
         <widget class="QTabWidget" name="tab_game_data">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="SearchFilterTreeView" name="attribute_list">
           <property name="expand_collapse_hidden" stdset="0">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Attributes</string>
           </attribute>
          </widget>
          <widget class="SearchFilterTreeView" name="skill_list">
           <property name="expand_collapse_hidden" stdset="0">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Skills</string>
           </attribute>
          </widget>
          <widget class="SearchFilterTreeView" name="facet_list">
           <property name="expand_collapse_hidden" stdset="0">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Facets</string>
           </attribute>
          </widget>
          <widget class="SearchFilterTreeView" name="preference_list">
           <attribute name="title">
            <string>Preferences</string>
           </attribute>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="script_tab">
      <attribute name="title">
       <string>Script</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="te_script">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="statusTip">
          <string>&lt;b&gt;&lt;font color=red&gt;WARNING: Any script set will be used in place of any attributes, skills, traits or preferences specified!!!&lt;/font&gt;&lt;/b&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;b&gt;&lt;font color=red&gt;WARNING: Any script set will be used in place of any attributes, skills, traits or preferences specified!!!&lt;/font&gt;&lt;/b&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QTextEdit" name="txt_status_tip">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Role changes will not be applied fully until the next read!!!</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="button_box">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_name">
       <property name="statusTip">
        <string>Select another dwarf to compare.</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_current">
       <property name="statusTip">
        <string>This is the current rating for this role. Note that this value is prior to comparison with the other dwarves.</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_new">
       <property name="statusTip">
        <string>This is the new rating for this role. Note that this value is prior to comparison with the other dwarves.</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_refresh_ratings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/img/refresh.png</normaloff>:/img/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Copy From</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbl_role_name">
       <property name="text">
        <string>Role Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="le_role_name">
       <property name="toolTip">
        <string>The name of the role.</string>
       </property>
       <property name="statusTip">
        <string>Creating a custom role with the same name as a default role will override it in the default Roles view.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cmb_copy">
       <property name="toolTip">
        <string>Target role to copy.</string>
       </property>
       <property name="statusTip">
        <string>Target role to copy.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_copy">
       <property name="toolTip">
        <string>Copy the selected role.</string>
       </property>
       <property name="statusTip">
        <string>Copies all weights and aspects from the selected role.</string>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="remove_action">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/minus-circle.png</normaloff>:/img/minus-circle.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="reset_default_weight_action">
   <property name="text">
    <string>Reset default weight</string>
   </property>
  </action>
  <action name="collapse_tree_aspects_action">
   <property name="text">
    <string>Collapse all</string>
   </property>
  </action>
  <action name="expand_tree_aspects_action">
   <property name="text">
    <string>Expand all</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchFilterTreeView</class>
   <extends>QWidget</extends>
   <header>searchfiltertreeview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>collapse_tree_aspects_action</sender>
   <signal>triggered()</signal>
   <receiver>tree_aspects</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>expand_tree_aspects_action</sender>
   <signal>triggered()</signal>
   <receiver>tree_aspects</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>RoleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>RoleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
